src/flatc.cpp

```diff
@@ -240,14 +234,6 @@ int FlatCompiler::Compile(int argc, const char **argv) {
         opts.allow_non_utf8 = true;
       } else if (arg == "--natural-utf8") {
         opts.natural_utf8 = true;
-      } else if (arg == "--no-js-exports") {
-        opts.skip_js_exports = true;
-      } else if (arg == "--goog-js-export") {
```

Comment:
How does this work now? I have a bunch of users internally who are setting this flag and it no longer is there.

Reply:
@bjornharrtell it is hard for me to see what changed here since it such a big merge. how does `--goog-js-export` work now that the flag is removed?

@aardappel

Reply:
Yes, it seems unwise to have removed that flag or any code related to it, since we couldn't test that externally. Apologies if I didn't spot that during this huge PR and huge review..

But this has been merged for a while now, so all we can do is try and fix this forward.

Reply:
Memory is vague at this point but I think the general idea was that TS is the only target and exports using es modules. If someone needs another module format they'd have to transpile it themselves.